tests/unit/test_units.py

```diff
@@ -139,6 +139,7 @@ def test_mach_tas(rng: np.random.Generator):
             "m_to_longitude_distance",
             "tas_to_mach_number",
             "mach_number_to_tas",
+            "dt_to_float_seconds",
```

Comment:
flag here if change the name

Reply:
5f96a0a772b5603c69292a3f9b7d781ef9b3114d